web/styles/message_row.css

```diff
@@ -667,6 +667,21 @@
     .message-edit-textbox {
         position: relative;
     }
+
+    &:has(.message_edit_content.over_limit),
+    &:has(.message_edit_content.over_limit:focus) {
+        box-shadow: 0 0 0 1pt hsl(0deg 76% 65%);
+    }
+
+    &:has(.message_edit_content.over_limit.flash) {
+        animation: message-limit-flash 0.5s ease-in-out 3;
+    }
+
+    &:has(.message_edit_content.invalid),
+    &:has(.message_edit_content.invalid:focus) {
+        border-color: hsl(3deg 57% 33%);
+        box-shadow: 0 0 2px hsl(3deg 57% 33%);
+    }
```

Comment:
Can you change these values to be CSS variables declared in `app_variables.css` and thus shared with the compose box code? Ideally the refactor for the existing code would be a tiny prep commit, and then this change would just be using those variables for its new CSS.

Reply:
Hm, apparently we don't really need those elements having class name `invalid`. I must have added it by mistake.